doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -21,6 +21,47 @@ This specification is licensed under the
 [FIWARE Open Specification License]
 (http://forge.fiware.org/plugins/mediawiki/wiki/fiware/index.php/FI-WARE_Open_Specification_Legal_Notice_%28essential_patents_license%29).
 
+## FIWARE NGSI information and data model introduction
+
+The FIWARE NGSI (Next Generation Service Interface) API defines 
+
+* a **data model** for context information, based on a simple information model using the notion of *context entities*.
+* a **context data interface** for exchanging information by means of query, subscription, and update operations.
```

Comment:
Fixed in https://github.com/tobiasjacobs/fiware-orion/commit/62031800a91478ad98241e15e0c18a4cd4e00318

(@tobiasjacobs , we use to mark that a given comment has been addressed in a given commit using a "Fixed in X" message as above. That way, specially for long PRs, is easier to trace new changes during the review. Just to let you know ;)).